src/socketio.quiver.js

```diff
@@ -115,23 +115,23 @@ QuiverSocialProvider.DEFAULT_SERVERS_ = [{
 }, {
   type: 'socketio',
   domain: 'dzgea1sj9ik08.cloudfront.net',
-  front: 'sdk.amazonaws.com'
+  front: 'js.arcgis.com'
 }, {
   type: 'socketio',
   domain: 'd2oi2yhmhpjpt7.cloudfront.net',
   front: 'api.mapbox.com'
 }, {
   type: 'socketio',
   domain: 'd2cqtpyb8m6x8r.cloudfront.net',
-  front: 'cdn.tinymce.com'
+  front: 'cdn.ckeditor.com'
 }, {
   type: 'socketio',
   domain: 'd3h805atiromvi.cloudfront.net',
   front: 'assets.tumblr.com'
 }, {
   type: 'socketio',
   domain: 'd2yp1zilrgqkqt.cloudfront.net',
-  front: 'www.splunk.com'
+  front: 'cdn.livefyre.com'
 }];
```

Comment:
Nice. Just created uProxy/uproxy#2651 so we don't forget about it. I'd love to help with that in any way I can.

![img](https://pbs.twimg.com/media/CpE_wRIXYAAHw9G.jpg)